### PR TITLE
PIM-8631: Fix column selector in the case of an attribute code as integer

### PR DESCRIPTION
**Pull request by pierallard**
| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | y
| Changelog updated                 | y
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: tests/legacy/features/pim/enrichment/product/datagrid/display_attributes_in_the_grid.feature [hunk 1]
```diff
@@ -54,3 +54,18 @@ Feature: Display product attributes in the grid
       | column    | title       |
       | Side view | **empty**   |
       | Top view  | akeneo2.jpg |
+
+  @jira https://akeneo.atlassian.net/browse/PIM-8631
+  Scenario: Successfully display attributes in the datagrid with code as integer
```
Comment (ahocquard) [inline tests/legacy/features/pim/enrichment/product/datagrid/display_attributes_in_the_grid.feature]: This is an edge case, we don't need an end-to-end test on it I think. Is a spec doable?
